modules/claims_api/app/sidekiq/claims_api/special_issue_updater.rb

```diff
@@ -53,13 +55,6 @@ def self.log_exception_to_claim_record(auto_claim_id, message)
       ClaimsApi::SpecialIssueUpdater.new.log_exception_to_claim_record(auto_claim_id, message)
     end
 
-    # Passthru to allow calling from sidekiq_retries_exhausted section above
-    #
-    # @param e [StandardError] Error to be logged
-    def self.log_exception_to_sentry(e)
```

Comment:
Verified this was not called in `sidekiq_retries_exhausted` & no class-level call to this method exists.